README.md

```diff
@@ -18,6 +19,56 @@ Specifically, if it enables services to scale _during_ deployment.
 > [!TIP]
 > A dashboard combining CloudFormation events, ASG activity and ASG metrics is available here - https://metrics.gutools.co.uk/d/cdvsv1d6vhp1cb/testing-asg-rolling-update.
 
+## Summary
+Can `AutoScalingRollingUpdate` be used to deploy updates to an auto-scaling group? Yes.
+- If not using horizontal scaling, `AutoScalingRollingUpdate` can be used, with any configuration.
+- If using horizontal scaling, `AutoScalingRollingUpdate` with dynamic configuration, is a viable deployment strategy. However, deployment times are significantly increased in certain conditions. 
+
+### What configuration should be used?
+The `AutoScalingRollingUpdate` policy has various [properties that can be set](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html#cfn-attributes-updatepolicy-rollingupdate).
+
+From our testing, we recommend the default configuration to be:
+- `MaxBatchSize` set to the max capacity of the autoscaling group
+- `MinActiveInstancesPercent` set to 100
+- `MinInstancesInService` conditionally set (see below)
+- `MinSuccessfulInstancesPercent` set to 100
+- `PauseTime` set to 5 minutes (the AWS default)
+- `SuspendProcesses` unset. Whilst AWS recommend suspending alarms, doing so would prevent scale-out events during deployment, which is the problem we're looking to solve.
```

Comment:
I'm slightly surprised by this one - I thought we had observed that this did not work as desired, so I presumed that we might suspend scaling actions for safety reasons (e.g. to avoid issues like [this](https://docs.google.com/document/d/1nWPDD4R4TzqIUe8iX5-oZfiV_bz3V5-JWX2p47AqamM/edit#bookmark=id.qnp0lptifhxt))?

Reply:
Agreed. I've updated now.